pkg/ebpf/telemetry/debugfs_test.go

```diff
@@ -14,7 +14,7 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-const testProfile = "./testdata/kprobe_profile"
+const testProfile = "../testdata/kprobe_profile"
```

Comment:
This is the only usage of this file, so I'd move it into this package.

Reply:
it is used for `ksyms_test.go` as well.
I'll split the test_data into 2 diff directories (one for parent ebpf pkg and one for the ebpf/telemetry pkg) in a separate PR

Reply:
The `kprobe_profile` file is not used for `ksyms_test.go`